charts/internal/calico/templates/typha/vpa.yaml

```diff
@@ -10,14 +10,14 @@ spec:
     kind: Deployment
     name: calico-typha-deploy
   updatePolicy:
-    updateMode: "Off"
+    updateMode: "Auto"
   resourcePolicy:
     containerPolicies:
     - containerName: "*"
       minAllowed:
-        memory: 100Mi
+        cpu: 100Mi
```

Comment:
@istvanballok Thanks for looking at it in detail. Indeed, it should be `cpu: 100m` instead of 100 cores.

Reply:
> instead of 100 cores.

Actually it was even a bit more: `100Mi`, i.e. about `104857600` CPU cores. :D

Just for my learning, I think [apimachinery/suffix.go](https://github.com/kubernetes/apimachinery/blob/a017454658b6eb672eb2d7b5eedad67f5f67665c/pkg/api/resource/suffix.go#L114) is resolving these suffixes here in [apimachinery/quantity.go](https://github.com/kubernetes/apimachinery/blob/a017454658b6eb672eb2d7b5eedad67f5f67665c/pkg/api/resource/quantity.go#L275-L288).